src/spawn_sync.cc

```diff
@@ -461,29 +462,29 @@ Maybe<bool> SyncProcessRunner::TryInitializeAndRunLoop(Local<Value> options) {
   uv_loop_ = new uv_loop_t;
   if (uv_loop_ == nullptr) {
     SetError(UV_ENOMEM);
-    return Just(false);
```

Comment:
These should probably not be changed.

Reply:
Ah right... yeah my eyes glazed right over the `Just(false)` return there.

Reply:
Actually no... these just should return `JustVoid()` instead of `Just(false)`.... where this method is called the only check that is made on the return value is `IsNothing()`